hr_birthday_welcome_message/static/src/js/greeting_message.js

```diff
@@ -0,0 +1,30 @@
+odoo.define("hr_birhday_welcome_message.greeting_message_birthday_message", function(
+    require
+) {
+    "use strict";
+
+    var core = require("web.core");
+    var greeting_message = require("hr_attendance.greeting_message");
+    var _t = core._t;
+
+    greeting_message.include({
+        init: function(parent, action) {
+            this._super.apply(this, arguments);
+            this.is_birthday = action.is_birthday;
+        },
+
+        _get_birthday_message: function() {
+            return _t(
+                "HAPPY BIRTHDAY! <i class='fa fa-birthday-cake' /> <br/> Enjoy your day"
```

Comment:
Maybe it is not necessary to be upper case, but just my opinion

Reply:
Changed :)